deploy/servicemesh-operator.yaml

```diff
@@ -10727,6 +10727,7 @@ spec:
         olm.relatedImage.v2_3.grafana: ${OSSM_GRAFANA_IMAGE}
         olm.relatedImage.v2_3.pilot: ${OSSM_PILOT_IMAGE}
         olm.relatedImage.v2_3.prometheus: ${OSSM_PROMETHEUS_IMAGE}
+        olm.relatedImage.v2_3.prometheus-config-reloader: ${OSSM_PROMETHEUS_CONFIG_RELOADER_IMAGE} 
```

Comment:
You should also add this image to `deploy/maistra-operator.yaml` and `deploy/src/deployment-maistra.yaml`.